modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco_{context}"]
+= Disabling Machine Config Operator from automatically rebooting
+
+When configuration changes are made by the Machine Config Operator, {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when `kube-apiserver-to-kubelet-signer` CA is rotated, or manual, such as when a registry or SSH key is updated, an {op-system} node reboots automatically unless is is  paused.
+
+To avoid unwanted disruptions, you can modify the machine config pool to prevent automatic rebooting after the Operator makes changes to the machine config.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have root access in {product-title}.
+
+.Procedure
+. To pause the autoreboot process after machine config changes are applied:
+
+* As root, modify the `spec.paused` field to `true`.
```

Comment:
should mention where this field is? inside the given MachineConfigPool crd (oc get mcp)

Reply:
I noticed slightly different behavior between the CLI and web console. In the web console, I could see the `PAUSED` field, but in the CLI output (`oc get mcp`), I could not. So I reworded this sentence to indicate that the field is in the MachineConfigPool CRD, but I didn't suggest running the command to see its output. (And because the rest of this procedure uses CLI and not web console, I didn't want to jump into console instructions here.)

Reply:
right, the MCP crd doesn't expose paused to the cli (maybe we should)